RockLib.Diagnostics.ConfigTests/Tracing/CustomTraceListener.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System;
 using System.Diagnostics;
+
+#pragma warning disable CA1050 // Declare types in namespaces
```

Comment:
[CA1050: Declare types in namespaces](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1050) - It says "It is safe to suppress a warning from this rule if performance is not a concern" so I'm suppressing it after spending time unsuccessfully adding it to a namespace without causing test failures.